sources/packages/backend/libs/test-utils/src/factories/student-restriction.ts

```diff
@@ -57,6 +61,8 @@ export async function saveFakeStudentRestriction(
     restrictionNote?: Note;
     resolutionNote?: Note;
     creator?: User;
+    updatedAt?: Date;
+    isActive?: boolean;
```

Comment:
`isActive` can be a factory input, but not under relations. should come from options.